tests/hwloc/ports/include/rsmi/rsmi.h

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright © 2013 Inria.  All rights reserved.
+ * Copyright (c) 2020, Advanced Micro Devices, Inc. All rights reserved.
+ * Written by Advanced Micro Devices,
+ * See COPYING in top-level directory.
+ */
+
+#ifndef HWLOC_PORT_RSMI_RSMI_H
+#define HWLOC_PORT_RSMI_RSMI_H
+
+/* we need to replace any RSMI-related #define that configure may have put in private/autogen/config.h */
+#ifndef HWLOC_CONFIGURE_H
+#error rsmi.h must be included after private/autogen/config.h
+#endif
+
```

Comment:
This file should be named exactly as the official headers used in topology-rsmi.c so I guess you should replace rsmi/rsmi.h with rocm_smi.h. That's likely the reason why our CI failed.

Reply:
Fix what I said above: the file should be in include/rsmi/rocm_smi.h (headers used to simulate rsmi build without rsmi are in include/rsmi/)
You may test this part with make check -C tests/hwloc/ports

Reply:
You mean it should be in tests/hwloc/ports/include/rsmi directory, but change file name to rocm_smi.h same as in topology-rsmi.c?

Reply:
Yes!